SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier1/X4.cs

```diff
@@ -176,5 +184,26 @@ public void RecalculateStatsEnd()
 
             private static void RecalculateStatsAPI_GetStatCoefficients(CharacterBody self, RecalculateStatsAPI.StatHookEventArgs args) { }
         }
+
+        public sealed class X4Buff : BaseBuffBehaviour
```

Comment:
This buff behaviour doesnt implement IBodyStatArgModifier, even then, This behaviour can be removed and instead use a direct call with RecalculateStatsAPI